synapse/models/inet.py

```diff
@@ -453,6 +453,138 @@ def _revModl201709181501(self):
+            ('inet:netuser:signup', 'inet:web:acct:signup'),
+            ('inet:netuser:signup:ipv4', 'inet:web:acct:signup:ipv4'),
+            ('inet:netuser:passwd', 'inet:web:acct:passwd'),
+            ('inet:netuser:seen:min', 'inet:web:acct:seen:min'),
+            ('inet:netuser:seen:max', 'inet:web:acct:seen:max'),
+
+            ('inet:netgroup:site', 'inet:web:group:site'),
+            ('inet:netgroup:name', 'inet:web:group:name'),
+            ('inet:netgroup:desc', 'inet:web:group:desc'),
+            ('inet:netgroup:url', 'inet:web:group:url'),
+            ('inet:netgroup:webpage', 'inet:web:group:webpage'),
+            ('inet:netgroup:avatar', 'inet:web:group:avatar'),
+
+            ('inet:netmemb:user', 'inet:web:memb:acct'),  # renamed from user -> acct
+            ('inet:netmemb:group', 'inet:web:memb:group'),
+            ('inet:netmemb:title', 'inet:web:memb:title'),
+            ('inet:netmemb:joined', 'inet:web:memb:joined'),
+            ('inet:netmemb:seen:min', 'inet:web:memb:seen:min'),
+            ('inet:netmemb:seen:max', 'inet:web:memb:seen:max'),
+
+            ('inet:follows:follower', 'inet:web:follows:follower'),
+            ('inet:follows:followee', 'inet:web:follows:followee'),
+            ('inet:follows:seen:min', 'inet:web:follows:seen:min'),
+            ('inet:follows:seen:max', 'inet:web:follows:seen:max'),
+
+            ('inet:netpost:netuser', 'inet:web:post:acct'), # renamed from netuser -> acct
+            ('inet:netpost:netuser:site', 'inet:web:post:acct:site'), # renamed from netuser -> acct
+            ('inet:netpost:netuser:user', 'inet:web:post:acct:user'), # renamed from netuser -> acct
+            ('inet:netpost:text', 'inet:web:post:text'),
+            ('inet:netpost:replyto', 'inet:web:post:replyto'),
+            ('inet:netpost:url', 'inet:web:post:url'),
+            ('inet:netpost:file', 'inet:web:post:file'),
+            ('inet:netpost:time', 'inet:web:post:time'),
+
+            ('inet:netfile:file', 'inet:web:file:file'),
+            ('inet:netfile:netuser', 'inet:web:file:acct'), # renamed from netuser -> acct
+            ('inet:netfile:netuser:site', 'inet:web:file:acct:site'), # renamed from netuser -> acct
+            ('inet:netfile:netuser:user', 'inet:web:file:acct:user'), # renamed from netuser -> acct
+            ('inet:netfile:name', 'inet:web:file:name'),
+            ('inet:netfile:posted', 'inet:web:file:posted'),
+            ('inet:netfile:ipv4', 'inet:web:file:ipv4'),
+            ('inet:netfile:ipv6', 'inet:web:file:ipv6'),
+            ('inet:netfile:seen:min', 'inet:web:file:seen:min'),
+            ('inet:netfile:seen:max', 'inet:web:file:seen:max'),
+
+        ]
+
+        def getXrefForms():
+            xreftypes = []
+            for tdef in self.core.getTypeDefs():
+                if tdef[1].get('subof') == 'xref':
+                    xreftypes.append(tdef[0])
+            return xreftypes
+
+        def updateXref(oldform, newform):
```

Comment:
These two functions should be moved into the store or core.

Reply:
After discussion, we are going to leave these functions in the migration until we decide how we want to handle cascading changes from xrefs.